src/components/vr_cooperation/src/register_service_request.cc

```diff
@@ -50,9 +52,32 @@ RegisterServiceRequest::~RegisterServiceRequest() {
 
 void RegisterServiceRequest::Execute() {
   LOG4CXX_AUTO_TRACE(logger_);
+
+  Json::Value params;
+  Json::Reader reader;
+  reader.parse(message_->json_message(), params);
+
+  int result_code;
+  bool success = true;
+  std::string info;
+  if (parent_->IsVRServiceSupported()) {
```

Comment:
I think checking support of service will be better to do in VRModule::ProcessMessage for all messages from mobile.